pandas/core/indexes/base.py

```diff
@@ -2316,16 +2316,31 @@ def shift(self, periods=1, freq=None):
 
     def argsort(self, *args, **kwargs):
         """
-        Returns the indices that would sort the index and its
-        underlying data.
+        Return the order of the indices.
```

Comment:
Hmm I think the "would sort" is important. Does "Return the order of the indices that would sort the index" sound OK and fit on a line?

Reply:
ok

Reply:
@TomAugspurger I think "order of the indices" is a bit strange no? I think it would be either "Return the order of the index" or "Return the indices that would sort the index", but not the combination